dev-util/distro-info/distro-info-1.0.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
No Python 3.9?

Reply:
No Python 3.9 in pylint

Reply:
It is [now](0e51dbc4000a3aa4dddb84ec3025640d22cd8f32)!